MrCMS.Web/Apps/Ecommerce/Services/Orders/OrderSearchService.cs

```diff
@@ -1,27 +1,60 @@
 ﻿using System;
-using Lucene.Net.Search;
-using MrCMS.Indexing.Querying;
+using MrCMS.Entities.People;
+using MrCMS.Helpers;
 using MrCMS.Paging;
-using MrCMS.Web.Apps.Ecommerce.Indexing;
 using MrCMS.Web.Apps.Ecommerce.Models;
-using MrCMS.Web.Apps.Ecommerce.Entities.Orders;
-using MrCMS.Web.Apps.Ecommerce.Services.Products;
+using NHibernate;
+using NHibernate.Criterion;
+using Order = MrCMS.Web.Apps.Ecommerce.Entities.Orders.Order;
 
 namespace MrCMS.Web.Apps.Ecommerce.Services.Orders
 {
     public class OrderSearchService : IOrderSearchService
     {
-        private readonly ISearcher<Order, OrderSearchIndex> _orderSearcher;
+        private readonly ISession _session;
 
-        public OrderSearchService(ISearcher<Order, OrderSearchIndex> orderSearcher)
+        public OrderSearchService(ISession session)
         {
-            _orderSearcher = orderSearcher;
+            _session = session;
         }
-        public IPagedList<Order> SearchOrders(OrderSearchModel model, int page = 1, int pageSize = 10)
+
+        public IPagedList<Order> SearchOrders(OrderSearchModel model)
         {
-            var query = new OrderSearchQuery(model);
-            return _orderSearcher.Search(query.GetQuery(), page, pageSize, query.GetFilter(),
-                new Sort(new SortField("id", SortField.INT, true)));
+            var queryOver = _session.QueryOver<Order>();
+
+            if (!String.IsNullOrWhiteSpace(model.SearchText))
+            {
+                User userAlias = null;

```

Comment:
Rather than joining onto the user here, use the OrderEmail and BillingAddress.LastName fields. 

This is because we support guest checkouts and imports from Amazon, which means that there will be orders created without a MrCMS User associated.